Tools/Parser/UnicodeData.cpp

```diff
@@ -69,6 +71,19 @@ UnicodeData::UnicodeData(const string &directory) :
             m_lastCodePoint = codePoint;
         }
 
+        std::string characterName;
```

Comment:
Please declare this string outside the loop. Also remove `std` prefix as it's already being used on top;

Reply:
Done. (Personally, I usually avoid `using namespace std;`, because it easily can cause name clashes.)